src/System.Net.Primitives/src/System/Net/Cookie.cs

```diff
@@ -2,10 +2,11 @@
 // The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
You should consider making equivalent changes to the copy of Cookie.cs here:
https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/src/netcore50/System/Net/cookie.cs

Long term....there won't be a copy of the file.  But due to the way that netcore50 needs cookie functionality, it needed to duplicate the file for now....could potentially be refactored to common code, etc.